public/index.html

```diff
@@ -43,12 +43,12 @@
     </nav>
 
     <!-- Dashboard---------------------------------------------------------------------------- -->
-    <section class="bg-light">
+    <section id="dashboard" class="bg-light">
         <div class="container py-5">
             <h3 class="py-3 fontHeading">Task Overview</h3>
             <div class="row">
                 <div class="col-md-4 col-12">
-                    <div class="card text-white mb-3" style="max-width: 18rem; background-color: #1976d2;">
+                    <span id="Business" type="button" class="card text-white mb-3" style="max-width: 18rem; background-color: #1976d2;">
```

Comment:
Please fix the span back to div this might create some problems with small screen, and type button is not needed it will work without that.

Reply:
k